src/topology/alexandroff.lean

```diff
@@ -0,0 +1,370 @@
+/-
+Copyright (c) 2021 Yourong Zang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yourong Zang, Yury Kudryashov
+-/
+import topology.separation
+import topology.opens
+
+/-!
+# The Alexandroff Compactification
+
+We construct the Alexandroff compactification of an arbitrary topological space `X` and prove
```

Comment:
```suggestion
We construct the Alexandroff compactification (the one-point compactification) of an arbitrary topological space `X` and prove
```
I know the other name is already in the tags, but I think this would make it clearer what this module is for.